woocommerce/payment-gateway/Frontend/Script_Handler.php

```diff
@@ -54,7 +54,20 @@ abstract class Script_Handler {
 	 */
 	protected function get_js_handler_class_name() {
 
-		return sprintf( '%s_5_6_1', $this->js_handler_base_class_name );
+		return sprintf( '%s_v5_6_1', $this->js_handler_base_class_name );
+	}
+
+
+	/**
+	 * Returns the JS event triggered after the JS handler class is loaded.
```

Comment:
```suggestion
	 * Gets the JS event triggered after the JS handler class is loaded.
```

should prefer `Gets` over `Returns`? I recall some discussion in the past about the verb to use - we voted for gets, but I can't find a reference in the wiki, sorry